static/js/compose.js

```diff
@@ -278,6 +278,10 @@ export function enter_with_preview_open() {
 }
 
 export function finish() {
+    if (compose_ui.compose_spinner_visible) {
```

Comment:
Let's add a comment documenting why we have this check.

Reply:
Possibly also including a bit of a docstring for `finish` -- I'd like it to be clear when reading this function that `finish` is the main entrypoint for sending a message via the compose box.

Reply:
Done